### PR TITLE
info on installing the latest stable version of Composer added

### PR DESCRIPTION
**Pull request by michalmilc90**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | master, 2.5, 2.4

Info on installing the latest stable version of Composer added.

### CODE REVIEW

File: docs/getting_started/requirements.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ These setups are tested by QA and are generally recommended setups. For security
 |Graphic Handler|GraphicsMagick or ImageMagick or GD|
 |[Clustering](../guide/clustering.md)|Linux NFS *or* S3/EFS *(for IO, aka binary files stored in content repository)*</br>Redis 3.2 or higher *(preferably separate instances for session & cache, both using one of the `volatile-*` [eviction policies](https://redis.io/topics/lru-cache))*</br>[Varnish](http://varnish-cache.org/) 5.1 or 6.0LTS *(recommended)* with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.rst) *or* [Fastly](https://www.fastly.com/) using [our bundle provided with eZ Platform Enterprise](../guide/http_cache.md#serving-varnish-through-fastly) *(for HttpCache)*|
 |Filesystem|Linux ext4 / XFS|
-|Package manager|Composer|
+|Package manager|Composer (latest stable version|
```
Comment (andrerom) [inline docs/getting_started/requirements.md]:
```suggestion
|Package manager|Composer (recent stable version|
```

File: docs/getting_started/install_ez_platform.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Before getting started, make sure you review the [requirements](requirements.md)
 
 ## Get Composer
 
-Install Composer, the PHP command line dependency manager.
+Install the latest stable version of Composer, the PHP command line dependency manager.
```
Comment (andrerom) [inline docs/getting_started/install_ez_platform.md]:
```suggestion
Install the recent stable version of Composer, the PHP command line dependency manager.
```

File: docs/getting_started/install_ez_platform.md [hunk 3]
```diff
@@ -24,13 +24,19 @@ Before getting started, make sure you review the [requirements](requirements.md)
...
 
-Install Composer, the PHP command line dependency manager.
+Install the recent stable version of Composer, the PHP command line dependency manager.
 Use the package manager for your Linux distribution. For example on Ubuntu:
 
 ``` bash
 apt-get install composer
 ```
 
+To check the version of Composer, run:
```
Comment (andrerom) [inline docs/getting_started/install_ez_platform.md]:
```suggestion
To verify you got a recent stable version of Composer, run:
```

File: docs/getting_started/install_ez_platform.md [hunk 4]
```diff
@@ -24,13 +24,19 @@ Before getting started, make sure you review the [requirements](requirements.md)
 
 ## Get Composer
 
-Install Composer, the PHP command line dependency manager.
+Install the recent stable version of Composer, the PHP command line dependency manager.
```
Comment (DominikaK) [inline docs/getting_started/install_ez_platform.md]:
```suggestion
Install a recent stable version of Composer, the PHP command line dependency manager.
```

File: docs/getting_started/install_ez_platform.md [hunk 5]
```diff
@@ -24,13 +24,19 @@ Before getting started, make sure you review the [requirements](requirements.md)
...
 Use the package manager for your Linux distribution. For example on Ubuntu:
 
 ``` bash
 apt-get install composer
 ```
 
+To verify you got a recent stable version of Composer, run:
```
Comment (DominikaK) [inline docs/getting_started/install_ez_platform.md]: Maybe place it in a tip box? It's not an obligatory step

File: docs/getting_started/requirements.md [hunk 6]
```diff
@@ -31,7 +31,7 @@ These setups are tested by QA and are generally recommended setups. For security
 |Graphic Handler|GraphicsMagick or ImageMagick or GD|
 |[Clustering](../guide/clustering.md)|Linux NFS *or* S3/EFS *(for IO, aka binary files stored in content repository)*</br>Redis 3.2 or higher *(preferably separate instances for session & cache, both using one of the `volatile-*` [eviction policies](https://redis.io/topics/lru-cache))*</br>[Varnish](http://varnish-cache.org/) 5.1 or 6.0LTS *(recommended)* with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.rst) *or* [Fastly](https://www.fastly.com/) using [our bundle provided with eZ Platform Enterprise](../guide/http_cache.md#serving-varnish-through-fastly) *(for HttpCache)*|
 |Filesystem|Linux ext4 / XFS|
-|Package manager|Composer|
+|Package manager|Composer (recent stable version|
```
Comment (DominikaK) [inline docs/getting_started/requirements.md]:
```suggestion
|Package manager|Composer (recent stable version)|
```

File: docs/getting_started/install_ez_platform.md (hunk 5 above)
Comment (michalmilc90) [reply docs/getting_started/install_ez_platform.md]: We would end up having two tip boxes one after another — if visually it's okay, then sure! :)

Comment (DominikaK) [reply docs/getting_started/install_ez_platform.md]: Fair enough. Or you could change it to `you can run:` to emphasise that the action is optional.

Comment (michalmilc90) [reply docs/getting_started/install_ez_platform.md]: Sure, no problem!

**Pull request merged**